src/irmin-pack/unix/control_file_intf.ml

```diff
@@ -180,7 +176,10 @@ module Payload_v4 = struct
 
       [chunk_start_idx] is the index for the starting chunk of the suffix
 
-      [chunk_num] is the number of chunks in the suffix *)
+      [chunk_num] is the number of chunks in the suffix
+
+      Renamed [suffix_end_poff] to [appendable_chunk_poff] to clarify this
```

Comment:
maybe move this comment inside the `control_file.ml`? it looks a bit weird to have a comment on renaming here, where there is no mention of `suffix_end_poff`.

Reply:
I was attempting to follow the pattern of referencing v3 and outlining what had changed. I updated the comment to try to make this connection more explicit. LMK what you think.

Reply:
Thanks, I think its better